pkgs/applications/networking/p2p/lokinet/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, fetchFromGitHub, cmake, libuv }:
+
+stdenv.mkDerivation rec {
+  pname = "lokinet";
+  version = "0.5.2";
+
+  bootstrap = fetchurl {
+    url = "https://seed.lokinet.org/bootstrap.signed";
+    sha256 = "1bsgs4r42fkfjk6wh60ml330h73rfgqdr623l3wcg983znlyzf6c";
```

Comment:
It does download with `lokinet-bootstrap`, I though this was something that didn't change all the time, so we could use it as part of the package instead to minimize something being downloaded from a url without hash checking.
I'll change it, as this is not the case.

Reply:
Yeah I really wasn't sure about this at the time. Thanks for clarifying.

If this file changes do we need to update it?.If so we could remove the `test` and just unconditionally download+update it. Whatever is required to keep the service running, I guess :smile:

Reply:
Hm, well having the package download the bootstrap would ensure we download it while there is internet access instead of have it download every service start/restart, so if you think that's safe, let's go for it.

Reply:
I'm not sure if that is safe or not... I don't know what the file is or does. If the file is important feels like that could be a problem, especially if the web site was hacked.

Will the file only change with application updates? Must the file always be up to date?

Reply:
https://github.com/loki-project/loki-network#running-on-linux
I'm not really sure, but from what I can see, it seems like it should be updated frequently.
Perhaps I should just call `lokinet-bootstrap` on each service start like before.

Reply:
Yeah if `lokinet-bootstrap` generates all needed files that would be ideal.